### DISCUSSION TITLE
Greek Language Leader

### DISCUSSION THREAD

**Discussion by gekoul** [OHF-Voice/intents#351] [Language Leaders]
Ready to contribute and assist on the Greek Language.

Working with HA for more than 2 years now.

**Comment by synesthesiam** [comment]
@dsimop - continuing Greek discussions here.

**Comment by pmentis** [comment]
I am willing to help with the Greek translation. @dsimop as a leader would you like to assign specific tasks?

**Comment by dsimop** [comment]
As said, would like to apply for the Greek Language Leader.
Software engineer / HA User for at least 3 years / Experienced with localisation projects (like the Greek HA Translations in Lokalise)
Already merged the first PR for initial EL Support. Moving forward with the next commits.

@pmentis sure, but that depends on the official assignment of the Greek language leader(s). We can coordinate for better efficiency.

**Comment by synesthesiam** [comment]
@dsimop Added you as Greek language leader :slightly_smiling_face:

**Comment by dsimop** [comment]
Hello everyone, I am happy to join the Greek team as a Greek Language Leader. As advised: "We require at least 2 contributions to consider a language leader application". So, by fulfilling the aforementioned requirement, you can also join as a Greek Language Leader.

The first two PRs have been successfully merged to master. Please have a look at the syntax of the .el files before starting with the translation process. And that, because we need to build the sentences in a general way by using the expansion rules. Still, as the recognition model is not adapted to the greek (and other languages), we proceed with a workaround. For more info, please have a look in the #236 conversation.

**Comment by dsimop** [comment]
So, most of the domain files have been successfully translated except from the last three ones (light related ones). In case anyone from the Greek Team wants to contribute, feel free to do so. Otherwise I will try to continue with that whenever I am available again.

Another point, we have to define precisely what kind of changes should be adapted into the recognition model by the PL in order not to have in the end 'weird' syntax in the name definitions.

**Comment by krasatos** [comment]
Hello everyone, Happy to help too if needed.

**Reply by dsimop** [reply]
Hello @krasatos, it would be definitely nice to contribute and make the greek community more active!
In order to be a language leader, you have to fulfil the requirements stated [here](https://developers.home-assistant.io/docs/voice/language-leaders) first.

**Comment by themiskir83** [comment]
Hi all, I would like to participate!

**Comment by synesthesiam** [comment]
@dsimop @themiskir83 @krasatos - would anyone be willing to translate the new responses into Greek?

**Reply by dsimop** [reply]
Resolved.

**Reply by synesthesiam** [reply]
Thanks! Greek is all green now: https://home-assistant.github.io/intents/

**Comment by FancyNerd92** [comment]
@dsimop I saw that we are behind in translated intents. How can I help with translation?

**Reply by dsimop** [reply]
You can set up the dev environment and contribute with the translations. I hope I can find the time this week to either contribute or check your PR.

**Comment by dsimop** [comment]
Hello everyone,
due to lack of time, I am not able to contribute as frequently as I want. Therefore, I would like to inform you I won't be the Greek Language Leader anymore. The position is now open and anyone interested on that can apply.

**Comment by chatziko** [comment]
Hello all, I'd like to apply for greek language leader. I'm involved in HA for a long time, made a few PRs in this repo and lots of other contributions in HA-related projects.

**Reply by dsimop** [reply]
Hello Kostas,
in case you are interested, this is the way to apply:
"If you want to apply to be a language leader, join us in #devs_voice on Discord or [open a discussion topic](https://github.com/home-assistant/intents/discussions)."
